src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -237,10 +237,14 @@ export default {
 				: 'Support a featured borrower with a microloan.';
 		},
 		featuredLoanTitle() {
-			return 'A loan ending soon';
+			return this.isLoggedIn
+				? `Recommended for you, ${this.firstName}`
```

Comment:
The first name text should be extracted into its own element and the class `data-hj-suppress` added to that new element.